ci/docker/install/ubuntu_r.sh

```diff
@@ -21,16 +21,16 @@
 # the whole docker cache for the image
 
 set -ex
+cd "$(dirname "$0")"
 # install libraries for mxnet's r package on ubuntu
 echo "deb http://cran.rstudio.com/bin/linux/ubuntu trusty/" >> /etc/apt/sources.list
 
-key=E084DAB9
-
-gpg --keyserver keyserver.ubuntu.com --recv-key $key || \
-    gpg --keyserver keyserver.pgp.com --recv-keys $key || \
-    gpg --keyserver ha.pool.sks-keyservers.net --recv-keys $key ;
-
-gpg -a --export $key | apt-key add -
+#key=E084DAB9
+#gpg --keyserver keyserver.ubuntu.com --recv-key $key || \
+#    gpg --keyserver keyserver.pgp.com --recv-keys $key || \
+#    gpg --keyserver ha.pool.sks-keyservers.net --recv-keys $key ;
+#gpg -a --export $key | apt-key add -
```

Comment:
I would just remove this and add a comment describing what r.gpg is and how to get it.

Reply:
I see your point, I think it's fine to leave that for documentation purposes. We can add a comment, but I think is overkill and expensive to go through another round of PR validation just for that. I can add it on a later change.